src/providers/deepinfra/chatComplete.ts

```diff
@@ -0,0 +1,151 @@
+import { DEEPINFRA } from "../../globals";
+import {
+  ChatCompletionResponse,
+  ErrorResponse,
+  ProviderConfig,
+} from "../types";
+
+export const DeepInfraChatCompleteConfig: ProviderConfig = {
```

Comment:
Can you please set the default, min and max values based on OpenAI docs.
https://platform.openai.com/docs/api-reference/chat/create

Reply:
The model default can still be of DeepInfra. If they do not have a default model mentioned, then we can pick up their most popular one or the one that they use in examples in their docs

Reply:
++update there was few endpoints using additional param's as per docs so add those as well.